filters/filters.go

```diff
@@ -247,3 +261,29 @@ func parseFiltersFile(filename string) ([]*Filter, error) {
 	}
 	return filters, nil
 }
+
+func ParseFiltersOpt(opt_str string) ([]*Filter, error) {
+	re := regexp.MustCompile("\\s*([^;]+);?")
+	var filters []*Filter
+	for _, m := range re.FindAllStringSubmatch(opt_str, -1) {
+		filter, err := parseFilterLine(m[1])
+		if filter != nil {
+			filters = append(filters, filter)
+			continue
+		} else if filter == nil && err == nil {
+			continue
```

Comment:
maybe just trim in the beginning of for loop? Instead of multiple trims and `nil, nil` return and `nil, nil` check
Do i miss something?

Reply:
you are right - I will change this part of the function